static/src/javascripts/projects/common/views/acquisitions-epic-viner-picture.html

```diff
@@ -0,0 +1,39 @@
+<div class="contributions__epic" data-component="<%=componentName%>">
+    <div>
+        <h2 class="contributions__title contributions__title--epic">
+            Since you’re here …
+        </h2>
+        <p class="contributions__paragraph contributions__paragraph--epic">
+            … we’ve got a small favour to ask. More people are reading the Guardian than ever, but far fewer are paying for it. Advertising revenues across the media are falling fast. And <span class="contributions__highlight">unlike many news organisations, we haven’t put up a paywall – we want to keep our journalism as open as we can</span>. So you can see why we need to ask for your help. The Guardian’s independent, investigative journalism takes a lot of time, money and hard work to produce. But we do it because we believe our perspective matters – because it might well be your perspective, too.
+        </p>
+        <p class="contributions__paragraph contributions__paragraph--epic">
+            If everyone who reads our reporting, who likes it, helps to support it, our future would be much more secure.
+        </p>
+        <p class="contributions__paragraph contributions__paragraph--epic">
+            <i>Katharine Viner</i>
+            <br>
+            <i>Editor-in-chief of the Guardian</i>
+
+        </p>
+    </div>
+
+    <div class="contributions__amount-field">
+        <div>
+            <a class="contributions__option-button contributions__contribute contributions__contribute--epic contributions__contribute--epic-member contributions__contribute--epic-member-top"
+               href="<%=membershipUrl%>"
+               target="_blank">
+                Become a supporter
+            </a>
+        </div>
+
+        <div>
+            <a class="contributions__option-button contributions__contribute contributions__contribute--epic contributions__contribute--epic-member"
+               href="<%=contributionUrl%>"
+               target="_blank">
+                Make a contribution
+            </a>
+        </div>
+
+    </div>
+    <img  class = "contributions__journalist_pic" src="https://i.guim.co.uk/img/static/sys-images/Guardian/Pix/contributor/2015/2/13/1423845167819/Katharine-Viner-R.png?w=300&q=55&auto=format&usm=12&fit=max&s=09cd84ed7ccabd5594e7a96bcf93d134">
```

Comment:
I'd recommend adding an `alt="Portrait Katharine Viner"` attribute or, because it's a less important ui element, using it as a background image (would make the positioning a bit easier).